Update description of SQL statements

andynava opened the pull request:
Specify that a row is "deleted" when the DELETE sql statement is used, instead of being "read".
Other consistency changes

azurecla commented:
Hi __@gbellmann__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

andynava committed:
Update description of SQL statements

Specify that a row is "deleted" when the DELETE sql statement is used, instead of being "read".
Other consistency changes
